cypress/TestCases/FireboltCertification/Advertising.feature

```diff
@@ -39,23 +39,23 @@ Feature: Advertising
    @Advertising @coreSDK @sdk @transport
    Scenario: Advertising.deviceAttributes - Positive Scenario: Get deviceAttributes
       When '3rd party app' invokes the 'Firebolt' API to 'get deviceAttributes'
-      Then 'Firebolt' platform responds with 'advertising device attributes'
+      Then 'Firebolt' platform responds for 'advertising device attributes'
 
    @Advertising @coreSDK @sdk @transport
    Scenario: Advertising.appBundleId - Positive Scenario: Get appBundleId
       When '3rd party app' invokes the 'Firebolt' API to 'get appBundleId'
-      Then 'Firebolt' platform responds with 'advertising appBundleId'
+      Then 'Firebolt' platform responds for 'advertising appBundleId'
 
    @Advertising @coreSDK @sdk @transport
    Scenario Outline: Advertising.advertisingId - Positive Scenario: Special Validation <Scenario>
       When '3rd party app' invokes the 'Firebolt' API to 'get advertisingId'
       And '3rd party app' invokes the 'Firebolt' API to 'get initialization parameters'
       When 1st party app invokes the 'Firebolt' API to '<API_Key>'
-      Then 'Firebolt' platform responds to '1st party app' with 'null for privacy setAllowAppContentAdTargeting'
+      Then 'Firebolt' platform responds to '1st party app' for '<API_Key>'
       When '3rd party app' invokes the 'Firebolt' API to 'get advertisingId'
       And '3rd party app' invokes the 'Firebolt' API to 'get initialization parameters'
-      Then 'Firebolt' platform responds with '<AdvertidingId_Validation_key>'
-      And 'Firebolt' platform responds with '<Parameters_Validation_key>'
+      Then 'Firebolt' platform responds for '<AdvertidingId_Validation_key>'
+      And 'Firebolt' platform responds for '<Parameters_Validation_key>'
```

Comment:
No need to change the glue to use **for** instead of **with** in case of getter

Reply:
updated